test/cases/tentative array.c

```diff
@@ -9,5 +9,4 @@ int arr[3]; // TODO should make the error go away
 #define EXPECTED_ERRORS \
     "tentative array.c:1:5: warning: tentative array definition assumed to have one element" \
     "tentative array.c:3:16: error: invalid application of 'sizeof' to an incomplete type 'int []'" \
-    "tentative array.c:5:5: error: redefinition of 'arr' with a different type" \
-    "tentative array.c:1:5: note: previous definition is here" \
+
```

Comment:
Should also remove the `TODO` and `SKIPPED_TESTS`.

Reply:
For this one the `warning: tentative array definition assumed to have one element` should not happen since the definition is later completed. I think we'd have to do something similar to what we do for `tentative_defs` in the Parser since we don't want to issue the warning until we're done and have determined the definition is incomplete.